ui/src/dashboards/components/Ranger.js

```diff
@@ -0,0 +1,99 @@
+import React, {PropTypes} from 'react'
+import _ from 'lodash'
+
+// TODO: add logic for for Prefix, Suffix, Scale, and Multiplier
+const Ranger = ({onSetRange, onSetLabel, axes}) => {
```

Comment:
Bhahaha.  

I love the name `Ranger` but it doesn't make sense anymore.  Since all of the options in here involve the `axes`, perhaps `AxesOptions` is a better name.